src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -64,103 +65,135 @@
         <service id="form.type.form" class="Symfony\Component\Form\Extension\Core\Type\FormType">
             <argument type="service" id="form.property_accessor" />
             <tag name="form.type" />
+            <deprecated>"The \"%service_id%\" service is deprecated since Symfony 3.1 and will be removed in 4.0."</deprecated>
```

Comment:
no we should not. We use lazy-loading for form type extensions to improve performance, which is why we don't register all form extensions